src/cgls/cgls.c

```diff
@@ -50,6 +51,7 @@ static void help(void) {
                "     --version        Show package version\n"
                "     --no-pager       Do not pipe output into a pager\n"
                "  -a --all            Show all groups, including empty\n"
+               "  -u --unit           Show the subtrees of specifified units\n"
```

Comment:
Yeah, I follow. It turned out a bit harder than I thought at the beginning, but in the end it works quite nicely, I think.